# This is the frequency of the original simulation recording
base_freq: 200
# An image will be captured every this many simulation ticks.
# The FPS of the camera is base_freq / camera_period
camera_period: 10
lidar_period: 20

should_do_depth: false
carla_hostname: localhost
carla_port: 2000

camera_params:
  fov: 90  # Field of view in degrees
  height: 480
  width: 720

# All transforms are in the coordinate frame expected by Rovio, im meters:
#  - x: Right
#  - y: Down
#  - z: Forward (along optical axis of camera)
# Rotations are in degrees:
#  - roll: Positive right (Around +z axis)
#  - pitch: Positive up (Around +x axis)
#  - yaw: Positive right (Around +y axis)

# Transform from vehicle frame to IMU0 frame (which is also the frame for Rovio)
imu0_transform:
  x: 0
  y: -1.5
  z: 1.5
  roll: 0
  pitch: 0
  yaw: 0

# Transform from vehicle frame to Camera frame
camera_transform:
  x: 0
  y: -1.5
  z: 1.5
  roll: 0
  pitch: 0
  yaw: 0

# Transform from vehicle frame to IMU1
imu1_transform:
  x: 0
  y: -1.5
  z: 1.5
  roll: 0
  pitch: 0
  yaw: 0

# Transform from vehicle frame to Lidar
lidar_transform:
  x: 0
  y: -1.5
  z: 1.5
  roll: 0
  pitch: 0
  yaw: 0